adapters/aidem/aidem.go

```diff
@@ -4,37 +4,49 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"text/template"
 
 	"github.com/prebid/openrtb/v19/openrtb2"
-	"github.com/prebid/prebid-server/v2/adapters"
-	"github.com/prebid/prebid-server/v2/config"
-	"github.com/prebid/prebid-server/v2/errortypes"
-	"github.com/prebid/prebid-server/v2/openrtb_ext"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/prebid/prebid-server/v2/macros"
 )
 
-type adapter struct {
-	endpoint string
+type AIDEMAdapter struct {
```

Comment:
Please keep this as `adapter`. There is no need to export the struct by starting with a capital letter. There's no reason to repeat the adapter name as it's already in the package name.